app/helpers/bulk_downloads_helper.rb

```diff
@@ -43,7 +43,7 @@ def get_valid_pipeline_run_ids_for_samples(samples)
 
   def format_bulk_download(bulk_download, detailed: false, admin: false)
     formatted_bulk_download = bulk_download.as_json(except: [:access_token])
-    formatted_bulk_download[:num_samples] = bulk_download.pipeline_runs.length
+    formatted_bulk_download[:num_samples] = bulk_download.pipeline_runs.empty? ? "NA" : bulk_download.pipeline_runs.length
```

Comment:
We should hide the "Number Samples" and "Samples in this Download" sections on the bulk download details sidebar UI if there are no pipeline runs. Instead of showing "NA".

Reply:
Great point, I made the change to hide the relevant sections. I do still need to handle the edge here in this line to prevent the "Samples" column from showing "0" in the main Downloads table (not Details sidebar). However, I changed my mind from "NA" to just leaving the field empty, how does that sound? (cc @happyimadesignr)

Reply:
![image](https://user-images.githubusercontent.com/16944816/80849018-1e2d9f00-8bb1-11ea-8dbb-56032c558df8.png)

![image](https://user-images.githubusercontent.com/16944816/80849030-2a196100-8bb1-11ea-99ce-415103a4bb8a.png)